src/main/kotlin/no/nav/familie/ef/sak/service/TilgangService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.ef.sak.service
+
+import no.nav.familie.ef.sak.api.ManglerTilgang
+import no.nav.familie.ef.sak.integration.FamilieIntegrasjonerClient
+import no.nav.familie.ef.sak.sikkerhet.SikkerhetContext
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class TilgangService(private val integrasjonerClient: FamilieIntegrasjonerClient,
+                     private val personService: PersonService,
+                     private val behandlingService: BehandlingService,
+                     private val fagsakService: FagsakService) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun validerTilgangTilPersonMedBarn(personIdent: String) {
+        val person = personService.hentPersonMedRelasjoner(personIdent)
+
+        integrasjonerClient.sjekkTilgangTilPersoner(person.identifikatorer()).forEach {
+            if (!it.harTilgang) {
+                throw ManglerTilgang("Saksbehandler ${SikkerhetContext.hentSaksbehandler()} har ikke tilgang til ${personIdent} eller dets barn")
+            }
+        }
+    }
```

Comment:
Det kan jo bli mange kall mot pdl+familie integrasjoner her. Burde vi cachet noe her, isåfall hva/hvordan løser vi det best?

Reply:
Kan vi se det an? Vet vi hva pdl mener? Hvor raskt må vi håndtere at noe endrer seg?

Reply:
Vi kan absolutt se det an, jeg tenker jo at ~15 minutters cache vil potensielt være "godt nok" for å slippe unna det verste

Reply:
Så lenge vi cacher kortere tid enn det tar å fikse en manglende tilganger eller liknende er det jo greit ;) 15min kunne kanskje vært ok. Tenker likevel dette kan være "neste oppgave"?

Reply:
Absolutt ikke noe som må gjøres nå, nei. Vil bare lufte tanken :D